TenureInformationApi/V1/Controllers/TenureInformationController.cs

```diff
@@ -50,7 +52,9 @@ public async Task<IActionResult> GetByID([FromRoute] TenureQueryRequest query)
         {
             var result = await _getByIdUseCase.Execute(query).ConfigureAwait(false);
             if (result == null) return NotFound(query.Id);
-            return Ok(result);
+
+            HttpContext.Response.Headers.Add(HeaderConstants.ETag, result.VersionNumber.Value.ToString());
```

Comment:
Shouldn't this be...
```
HttpContext.Response.Headers.Add(HeaderConstants.ETag, $"\"{result.VersionNumber.Value}\"");
```

Reply:
or `new EntityTagHeaderValue($"\"{result.VersionNumber.Value}\"", false).ToString()`

Reply:
I've just tried Mike's suggestion and it looks like it has worked!